library/src/scala/PolyFunction.scala

```diff
@@ -1,10 +1,26 @@
 package scala
 
+/* (EXPERIMENTAL) NEW DOCUMENTATION: This should be used when we stabilize erased definitions or other generalize other function types.
+ *
+ * Marker trait for many (poly) function types.
+ *
+ *  This is the only trait that can be refined with a method or polymorphic method,
```

Comment:
This isn't the only trait that can be refined with a method (any trait can as long as it's not polymorphic) so the documentation needs to be reworded.